### ISSUE TITLE
 SteamID.fromIndividualAccountID('') says it's valid

### ISSUE THREAD

**Issue by busheezy** [DoctorMcKay/node-steamid#3]
```js
const SteamID = require('steamid');

const steamID = SteamID.fromIndividualAccountID('');

console.log('valid', steamID.isValid());
```

This probably shouldn't return as valid.

Thanks.

**Comment by DoctorMcKay** [comment]
Fixed in v1.1.1, thanks.

**Comment by busheezy** [comment]
Thank you!
